app/helpers/orders_helper.rb

```diff
@@ -0,0 +1,44 @@
+module OrdersHelper
+
+  def final_total(array)
+    total = 0
+
+    array.each do |item|
```

Comment:
This belongs in the model as it's doing business logic.  View helpers are concerned with display.   Minor issue however.